include/libtorrent/string_view.hpp

```diff
@@ -41,11 +41,13 @@ POSSIBILITY OF SUCH DAMAGE.
 #include <boost/utility/string_ref.hpp>
 namespace libtorrent {
 using string_view = boost::string_ref;
+using wstring_view = boost::wstring_ref;
```

Comment:
ignore my comment then, just saw `std::basic_string_view<wchar_t>` in my search but not the alias type.

looking forward for the big move to `std::error_code`!! (and maybe use `asio` without boost)